README.md

```diff
@@ -1 +1,65 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+This project is to set up a mobile version skeleton of my portfolio. The goal is to show understanding of the
+- css flexbox module
+- Understand how to parse a Figma design to create a UI.
+- Build a personal portfolio site.
+- Use images and backgrounds to enhance the look of the website.
+- Requesting code review
+- Requesting PR
+## Built With
+- Html
+- Css
+- Flexbox Module
+## Live Demo
+- https://mansa-susa.github.io/microverse_hello/
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Html
+- Css
+### Setup
+Clone repository from [@githubhandle](https://github.com/mansa-susa)
+
+Open files in your Text editor / IDE
+
+Load on Browser
+
+### Install
+
+### Usage
+
+### Run tests
+Linters 
+- Lighthouse
+- Webhint (run npx hint .)
+- Stylelint (run npx stylelint "**/*.{css,scss}" )
+
+### Deployment
+
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] README file is almost there! Let's add your actual name here, this way people will know how to address you.

Reply:
Thanks All resolved